meta-smarcimx8m-extras/recipes-alexa/avs-device-sdk/avs-device-sdk_%.bbappend

```diff
@@ -0,0 +1,2 @@
+# build SDK without snowboy support as it doesn't provide aarch64 binaries for Linux
```

Comment:
This (and the commit message) still reads wrong to me. There are Linux aarch64 binaries (`aarch64` in the name of the folders in snowboy) but you get errors when you try to use them.

Reply:
Comment here was not changed?

Reply:
forgot to git add :man_facepalming: